scripts/azure_test_image_version.ps1

```diff
@@ -0,0 +1,45 @@
+param (
```

Comment:
This is not yet coded as we discussed.

Reply:
- The resource group we are accessing as env variable 
- TestVersion contains "-" so we cannot able to access via env variable
- deploymentOutput also we can't able to use env variable since it is ouput variable